scripts/deploy.js

```diff
@@ -0,0 +1,13 @@
+const exec = require('child_process').exec;
+
+exec('npm publish', (error, stdout, stderr) => {
```

Comment:
This script is an artifact of basing everything on the [buy button's deployment](https://github.com/Shopify/buy-button-js/blob/master/script/deploy.js).  My intent was to create a standard structure to guide the eventual shipit/npm integration.  

Now that you mention it, this does seem like overkill, so I think I'll revise my PRs to use `"deploy": "npm publish"`.